var/spack/repos/builtin/packages/python/package.py

```diff
@@ -17,6 +20,16 @@
 from spack.util.prefix import Prefix
 
 
+# TODO: can be replaced with contextlib.redirect_stdout once we drop Python 2.7 support
```

Comment:
We could add this to the new `compat` module that @cosmicexplorer is working on in #21720.